src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1026,6 +1052,7 @@ def _bigbird_block_rand_mask_with_head(
             num_heads: int. total number of heads.
             plan_from_length: list. plan from length where num_random_blocks are choosen from.
             plan_num_rand_blocks: list. number of rand blocks within the plan.
+            indices_prng_key: jax.random.PRNGKey. PRNG key that is used to perform random jax operations
```

Comment:
Same again here just for consistency
```suggestion
            indices_prng_key: jax.random.PRNGKey. PRNG key that is used to perform random jax operations.
```